core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -1121,11 +1121,13 @@ public void testAllParamsHex() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
What's the reason to ignore this test and the following one?

Reply:
this is Alex hunting for code that broke test coverage